### ISSUE TITLE
Multiple files scraping problem when movie not found in Data18 Movie scraper

### ISSUE THREAD

**Issue by 12floz355** [DoctorD1501/JAVMovieScraper#55]
I made a comment on a "closed" issue that the problem that had been fixed was still not working, at least for the Data18 Movie scraper.  I decided to just open a new issue for it.  The below info is copy and pasted from the original issue's poster, so I hope that is ok.

I noticed this issue while scraping multiple files at once.

For example if the program scrapes 5 files and one of them cannot be found, it will write the file data on the wrong folder/file and skipping the last one.

e.g.

If i were to scrape the following files (only 1 is wrongly named/doesn't exist)

SOE-101
SOE-200
SOE-9999 (The wrong/non-existant one)
SOE-502
SOE-503

After scraping and clicking on write file data, it will scrape like this :

SOE-101 (Scraped with the correct info) >
SOE-200 (Scraped with the correct info) >
SOE-9999 (Scraped with the info of SOE-502) >
SOE-502 (Scraped with the info of SOE-503) >
SOE-503 (no written info at all due to SOE-999 not found)

**Comment by DoctorD1501** [comment]
Should be fixed now in 0.1.21-alpha. Just let me know if you're still having this issue. Thanks.

**Comment by 12floz355** [comment]
Hi thanks for the fix but it is only partially working.  I tested two different sets of three tittles.  The first set had a movie that was found, then a movie that was not found AND did not bring up the selection box but just moved on to the next movie that was found.  This combination was able to write the files out properly.  

The second set was the same as above but the middle movie that it could not find a good match for did bring up the selection box which I then hit cancel for (also tried once where I did not select a movie from the list and hit OK instead of cancel).  This combination had the original problem.
